tests/test_examples.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+import concepts
+
+
+def test_load_dataset():
+    context = concepts.load_dataset('livingbeings_en')
```

Comment:
We should probbaly not depend on internet connectivity in the test.

How about having a mocked test and another one doing the actual thing that would be opt-in with a flag?

Something mildy related here: https://github.com/xflr6/graphviz/blob/e5578d39009469df2b7c6743458970643e228226/tests/conftest.py#L5

Reply:
I agree but I did not manage to implement this. The examples I found are mainly targeting ~unittest~ and not ~pytest~ (or ~pytest~ with the ~requests~ module).